hotosm/tasking-manager
# Licenses frontend

Closes #2609 
**How to test**
1. As an admin create a license.
2. With a created project, associate a project with the license by going to the project settings in the imagery section.
3. Then, try to map a task inside the project. You will receive a message with the license and the option to accept or cancel.
4. When accepted, you will be redirected to the editor to map the task.
5. Try to map another task. The popup message will not appear, since the licenses has been accepted by the user
